cmd/openshift-install/gather.go

```diff
@@ -197,3 +202,34 @@ func unSupportedPlatformGather(directory string) error {
 
 	return logGatherBootstrap(gatherBootstrapOpts.bootstrap, 22, gatherBootstrapOpts.masters, directory)
 }
+
+func logClusterOperatorConditions(ctx context.Context, config *rest.Config) error {
+	client, err := configclient.NewForConfig(config)
+	if err != nil {
+		return errors.Wrap(err, "creating a config client")
+	}
+
+	operators, err := client.ConfigV1().ClusterOperators().List(metav1.ListOptions{})
+	if err != nil {
+		return errors.Wrap(err, "listing ClusterOperator objects")
+	}
+
+	for _, operator := range operators.Items {
+		logger := logrus.WithField("operator", operator.ObjectMeta.Name)
+		for _, condition := range operator.Status.Conditions {
+			if (condition.Type == configv1.OperatorAvailable || condition.Type == configv1.OperatorUpgradeable) && condition.Status == configv1.ConditionTrue {
+				continue
+			}
+			if (condition.Type == configv1.OperatorDegraded || condition.Type == configv1.OperatorProgressing) && condition.Status == configv1.ConditionFalse {
+				continue
+			}
+			logger.WithFields(logrus.Fields{
+				"condition": condition.Type,
+				"status":    condition.Status,
+				"reason":    condition.Reason,
+			}).Info(condition.Message)
```

Comment:
I think this is an ERROR level, since we are not providing information, but summarizing errors

Reply:
We want to show `Progressing=True`, and that's not strictly an error.  And my [local test][1] had a `Disabled=False`, which sounds fine too, I just print it because Disabled is not [a well-known type][2].  So I dunno how much customization we want here.  Would it be enough to use Error for Degraded and Info for everything else?

[1]: https://github.com/openshift/installer/pull/2450#issuecomment-537728581
[2]: https://github.com/openshift/api/blob/d92789481b059132f1febccd9bb47cb27220f248/config/v1/types_cluster_operator.go

Reply:
> Would it be enough to use Error for Degraded and Info for everything else?

Done with 7c7e4cf -> 618a84f.